java/maven/src/org/netbeans/modules/maven/options/SettingsPanel.java

```diff
@@ -826,7 +850,7 @@ public void valueChanged(javax.swing.event.ListSelectionEvent evt) {
                 .addComponent(lblCategory)
                 .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)
                 .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
-                    .addComponent(pnlCards, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
+                    .addComponent(pnlCards, javax.swing.GroupLayout.DEFAULT_SIZE, 547, Short.MAX_VALUE)
```

Comment:
Can you switch this back to `DEFAULT_SIZE`? This would set the component to a fixed size and layout managers won't resize it.

When I open the options now i see scroll bars even though everything would fit into the window.

To achieve this its often possible to open the panel in question via double click on the navigator. Then reduce the size till it snaps to default size.

Reply:
argh ... stupid copy / cleanup error before commit: missing assignment. Thanks for catching.